src/tactic/interactive.lean

```diff
@@ -1083,5 +1083,13 @@ add_tactic_doc
   decl_names := [`tactic.interactive.generalize'],
   tags       := ["context management"] }
 
+/--
```

Comment:
I don't like docstrings starting with "Like another definition, but ..." since I have to click through to see what it actually does. So let's copy the `subst` docstring:
```suggestion
/--
Given hypothesis `h : x = t` or `h : t = x`, where `x` is a local constant,
`subst' h` substitutes `x` by `t` everywhere in the main goal and then clears `h`.

```